### ISSUE TITLE
Native Snapshots Overwrite Web Snapshots

### ISSUE THREAD

**Issue by gardner** [VISI-ONE/create-react-native-web-app#2]
**Describe the bug**
the `switchSnapshots()` method copies files from the `__snapshots__` folder into the __web_snapshots__ folder which makes sense if the snapshots folder has been updated. Unfortunately, running the web tests and then the native tests back to back causes one to override the other. It appears to be happening here: https://github.com/VISI-ONE/rnw-starter-app/blob/master/scripts/switchSnapshots.js#L25

**To Reproduce**
1. `npm install -g create-react-native-web-app`
2. `create-react-native-web-app myApp`
3. `cd myApp`
4. Generate snapshots: `npm run test:web -- -u && npm run test:native -- -u`
5. `npm run test:web`
6. `npm run test:web`
7. `npm run test:native`
8. Please note the tests fail due to incorrect snapshots.

**Expected behavior**
The tests to pass

**Screenshots**
![screenshot from 2018-07-13 10-28-44](https://user-images.githubusercontent.com/787026/42662969-ba93580c-8687-11e8-8a19-09470175dc4d.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 16.04 LTS
 - Browser Chrome
 - Version 67.0.3396.99 (Official Build) (64-bit)

**Smartphone (please complete the following information):**
 - Device: N/A
 - OS: N/A
 - Browser N/A
 - Version N/A

**Additional context**
Thanks for making this project!

**Comment by gardner** [comment]
Ahh thanks for that. I must have misunderstood the purpose of swapping snapshots into and out of the __snapshots__ directory. What is the proposed workflow for testing these kinds of apps? Why do the scripts copy files into and out of the __snapshots__ directory?

**Comment by gardner** [comment]
Perhaps this could help: https://github.com/facebook/jest/pull/6143

**Comment by orYoffe** [comment]
Yep thanks @gardner , that would help but since it's still not merged and i didn't find a different solution.. I had to solve the issue of web and native deleting each others snapshots and keep them also in the repo to be committed.

after it's merged i would love to throw away the switchSnapshots script and implement the snapshotResolver solution :)

**Comment by gardner** [comment]
I am still confused about the testing workflow for testing these kinds of apps. Why do the scripts copy files into and out of the __snapshots__ directory?

**Comment by orYoffe** [comment]
@gardner Jest is using the __snapshots__ folder to check the snapshots (obsolete or not) so in order to use them i keep a copy in the other dedicated folders and place the env snapshots in the __snapshots__ so the tests for the env you're running would work...

you can try disabling the script and see what happens and maybe you'll find a better solution :)
